src/components/car-form/index.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import * as util from '../../lib/utilities.js';
+
+class CarForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state {
+      make: '';
+      model: '';
+      color: '';
+      licensePlate: '';
+    }
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e) {
+  let {name, value,} = e.target;
+  this.setState({
+    [name]: value
+  })
```

Comment:
since one of the inputs accepts a number we will have to capture it like this:
```
    if(type === 'number') {
      try {
        this.setState({
          [name]: parseInt(value)
        })
      } catch(err) {
        console.log(err);
      }
    } else {
      this.setState({
        [name]: value
      })
    }
```